trainer/derrian_backend/custom_scheduler/LoraEasyCustomOptimizer/low_bit_optim/cpu_offload.py

```
@@ -0,0 +1,184 @@
+    def state_dict(self):
+        state_dict = {
+            "offloaded": [optim.state_dict() for optim in self.optim_dict.values()]
+        }
+        if self.d_opt:
+            state_dict["on-device"] = self.d_opt.state_dict()
+        return state_dict
+
+    def load_state_dict(self, state_dict):
+        for optim, optim_state_dict in zip(
+            self.optim_dict.values(), state_dict["offloaded"]
+        ):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)